apps/researcher/src/app/[locale]/objects/[id]/metadata.tsx

```diff
@@ -39,48 +37,48 @@ export function MetadataContainer({identifier, children}: Props) {
 
 interface MetadataEntryProps {
   children: ReactNode;
-  isOwner: boolean;
+  isCurrentLocation: boolean;
```

Comment:
Is the 'location' the place where the heritage object is being held? If so, this is not in the data that we currently have. But we do know the publisher of the data about the heritage object. Replace with e.g. 'isCurrentPublisher'?